gapic-generator-java/src/test/java/com/google/api/generator/gapic/composer/ComposerTest.java

```diff
@@ -53,8 +56,13 @@ class ComposerTest {
           .build();
   private List<Sample> ListofSamples = Arrays.asList(new Sample[] {sample});
 
+  @Before
+  public void initialSanityCheck() {
+    assertTrue(context.containsServices());
+  }
+
   @Test
-  void gapicClass_addApacheLicense() {
+  public void gapicClass_addApacheLicense_validInput_succeeds() {
```

Comment:
Could you remove the `public` in the signature?

Reply:
Done